### PR TITLE
@orta => fix more info

### PR DESCRIPTION
**Pull request by jorystiefel**

### CODE REVIEW

File: Artsy_Tests/View_Controller_Tests/Artwork/ARArtworkInfoViewControllerTests.m [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    });
+    
+    it(@"displays correctly when missing section", ^{
+        
+        Artwork *artwork = [Artwork modelWithJSON:@{
+                                                    @"provenance" : @"Ancient Hoboken",
+                                                    @"additional_information" : @"The more you know",
+                                                    @"literature": @"Books About Art",
+                                                    @"exhibition_history" : @"Metro Pictures April 2014"
+                                                    }];
```
Comment (orta) [inline Artsy_Tests/View_Controller_Tests/Artwork/ARArtworkInfoViewControllerTests.m]: can you please move these to be 1 indent next to the JSON e.g:

```
    Artwork *artwork = [Artwork modelWithJSON:@{
        @"provenance" : @"Ancient Hoboken",
        @"additional_information" : @"The more you know",
        @"literature": @"Books About Art",
        @"exhibition_history" : @"Metro Pictures April 2014"
    }];
```

**Comment by jorystiefel** [comment]
Fixes #130

**Comment by jorystiefel** [comment]
@orta everything look correct here?

**Comment by orta** [comment]
MRW when I receive reddit gold but have no idea what it does

FAB

**Comment by alloy** [comment]
@jorystiefel Nice :+1: Can you create a new ticket, and work on it, about the fact that Markdown in these texts are not being rendered (as shown in my screenshots on #130)?

**Pull request merged**